akka-docs/rst/java/camel.rst

```diff
@@ -281,7 +281,7 @@ Actors may also use a Camel `ProducerTemplate`_ for producing messages to endpoi
 
 .. includecode:: code/docs/camel/MyActor.java#ProducerTemplate
 
-For initiating a a two-way message exchange, one of the
```

Comment:
In the IntelliJ IDEA find dialog, this regular expression 
`\b(\w+)\s+\1\b` 
catches all consecutive duplicate words in `.rst` files.